iati/core/tests/test_resources.py

```diff
@@ -1,6 +1,7 @@
 """A module containing tests for the library implementation of accessing resources."""
 from lxml import etree
 import pytest
+from six import string_types
```

Comment:
It's possible by either modifying the test code from `assert isinstance(result, string_types)` to

```
assert (isinstance(result, str) or  # Python3
        isinstance(result, unicode))  # Python2
```

This will fail linting under Python3 though, since `unicode` isn't a type there...

Other alternatives involve following one of the less accepted answers in [this SE post on the issue](https://stackoverflow.com/questions/11301138/how-to-check-if-variable-is-string-with-python-2-and-3-compatibility)

Do we know where `six` might cause problems?